schedule/schedule-appengine/src/main/client/jasify/admin/organization/admin-organization.js

```diff
@@ -43,11 +43,29 @@
         vm.addActivityType = addActivityType;
         vm.removeActivityType = removeActivityType;
 
+        vm.cashSet = false;
+        vm.payPalSet = false;
+
```

Comment:
I would have preferred if the solution would scale when we added a new paymentType Enum.   This way when we add another payment type, we need to revisit this page.